arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -1181,7 +1181,7 @@ ::frame_image( )
 
       AVFrame rgb_frame;
       av_image_fill_arrays(rgb_frame.data, rgb_frame.linesize,
-                           (uint8_t*)d->current_image_memory->data(),
+                           static_cast<uint8_t*>(d->current_image_memory->data()),
```

Comment:
Given I don't think it belongs anyway, I'm going to pretend I don't see that there is no space before the `*` here, while there is elsewhere :stuck_out_tongue:.